packages/components/src/components/overflow-menu/_overflow-menu.scss

```diff
@@ -65,6 +65,12 @@
     box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.3);
   }
 
+  .#{$prefix}--overflow-menu--light.#{$prefix}--overflow-menu--open,
+  .#{$prefix}--overflow-menu--light.#{$prefix}--overflow-menu--open
```

Comment:
any reason for the duplicate selector?

Reply:
the second line doesn't have a comma so it goes with the third line